securedrop_client/gui/widgets.py

```diff
@@ -3671,13 +3716,63 @@ def __init__(self, source: Source, controller: Controller) -> None:
         separator_font.setLetterSpacing(QFont.AbsoluteSpacing, 2)
         separator_font.setBold(True)
 
+        download_section = self.addSection(_("DOWNLOAD"))
```

Comment:
Thanks for bringing this back to my attention out of band, @gonzalo-bulnes.  It looks like gettext provides functions for ["using contexts for solving ambiguities"](https://www.gnu.org/software/gettext/manual/html_node/Contexts.html).  We don't currently use contexts anywhere, but it would be easy enough to replace our `_()` alias with a wrapper function to deal with both absolute and contextual message IDs (here, `DOWNLOAD`).